app/controllers/miq_ae_class_controller.rb

```diff
@@ -1822,11 +1825,14 @@ def create_action_url(node)
 
   def get_rec_name(rec)
     column = rec.display_name.blank? ? :name : :display_name
-    if rec.kind_of?(MiqAeNamespace) && rec.domain? && (!rec.editable? || !rec.enabled)
-      add_read_only_suffix(rec, rec.send(column))
-    else
-      rec.send(column)
+    if rec.kind_of?(MiqAeNamespace) && rec.domain?
+      editable_domain = User.current_tenant.editable_domains.include?(rec)
```

Comment:
so "is this editable" is an authorization scheme. ala cancan. This should be external, much like rbac search is external.

a) tenant should not even have "editable domains"... but I'm trying to pick my fights here.
b) this should read something like `can?(:edit, rec)`